jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -139,7 +139,5 @@ def _run_functions(self, *args, **kwargs):
                 temp_result = FUNCTIONS[function](self._spectrum1d)
 
             temp_results.append({'function': function, 'result': str(temp_result)})
-            self.result_available = True
-
-            self.results = []
```

Comment:
I've seen this elsewhere with a comment that its to trick the traitlets into forcing an update... is it ok to remove here?  Can we remove it everywhere?